web/approval.go

```diff
@@ -108,7 +112,7 @@ func approvePullRequest(c context.Context, params HookParams, id int, pullReques
 
 	if setStatus {
 		if !approval.AuthorAffirmed {
-			if params.Event == "pull_request" && affirmMsgActions.Contains(params.Action) {
+			if params.Approval != nil && params.Approval.IsApproval(&request) {
```

Comment:
So question: is this going to write a comment for every approval when there isn't an author approval?

Reply:
If N approvals are required (ignoring the author affirm feature) then the comment will appear when the Nth approval is received.